client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx

```diff
@@ -5,29 +5,89 @@
  */
 
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import { localize } from 'i18n-calypso';
+import Badge from 'components/badge';
 
 class GoogleAppsProductDetails extends Component {
 	static propTypes = {
 		annualPrice: PropTypes.string.isRequired,
 		monthlyPrice: PropTypes.string.isRequired,
+		showDiscount: PropTypes.bool.isRequired,
 	};
 
-	render() {
+	static defaultProps = {
+		showDiscount: false,
+	};
+
+	renderPrice() {
 		const { translate } = this.props;
 
+		if ( this.props.showDiscount ) {
+			return (
+				<Fragment>
+					<del>
+						{ translate( '%(monthlyPrice)s per user /month', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '%(monthlyPrice)s per user / month' )` *ES Score: 23*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).